zpm.c

```diff
@@ -260,18 +275,21 @@ int plugins_update_local_clone() {
     char* listing = get_zpm_plugin_list();
     char* plugin_name = strtok(listing, "\n");
 
-    if(!strcmp(listing, "Nothing to show.")) {
+    if (!strcmp(listing, "Nothing to show.")) {
       printf("Nothing to update.");
       return -1;
     }
     printf("Updating plugins ...\n");
     while(plugin_name) {
-      strcpy(command, "cd ~/.zpm/plugins/");
-      strcat(command, plugin_name);
-      strcat(command, "; git pull");
-      printf("Updating %s...\n", plugin_name);
-      ret = system(command);
-      plugin_name = strtok(NULL, "\n");
+        if (plugin_name[0] == '/')
+            goto next;
```

Comment:
This 'goto' could be translated to a simple 'if', or is there a reason to use 'goto'?